packages/roosterjs-react-editor/lib/utils/createEditorViewState.ts

```diff
@@ -1,8 +1,9 @@
 import EditorViewState from '../schema/EditorViewState';
+import { convertInlineCss } from 'roosterjs-editor-dom';
 
 export default function createEditorViewState(initialContent?: string): EditorViewState {
```

Comment:
Should we also accept `additionalStyleNodes` as an optional parameter, so that we can pass it down to `convertInlineCss`

Reply:
let's that one when need it